README.md

```diff
@@ -63,11 +63,10 @@ with any name. We recommend that you
 ### In Unix
 The steps to create and use virtual environments in Unix are:
 
-7. Create an environment by typing ``./createnv.sh `echo $0` <name>``.
+7. Create an environment by typing ``./createnv.sh <name>``.
```

Comment:
Here and elsewhere: the double backticks can now become a single backtracks because the inner backpack of the echo command has gone.